src/DIRAC/Resources/MessageQueue/Simple/StompInterface.py

```diff
@@ -0,0 +1,406 @@
+            )
+
+        print(f"WARNING: deprecated do not give the full mqURI, just the service name: {mqService}")
+        mqService = mqService.split("::")[0]
+
+    brokerParams = returnValueOrRaise(gConfig.getOptionsDict("/Resources/MQServices/%s" % mqService))
+    # This is for compatibility reasons with the existing configuration definition
+    # Although there are no reasons to separate queues and topics for stomp
+    topics = [
+        f"/topic/{dest}" for dest in gConfig.getSections("/Resources/MQServices/%s/Topics" % mqService).get("Value", [])
+    ]
+    queues = [
+        f"/queue/{dest}" for dest in gConfig.getSections("/Resources/MQServices/%s/Queues" % mqService).get("Value", [])
+    ]
+
+    brokerParams["destinations"] = topics + queues
+
+    return brokerParams
+
+
+@convertToReturnValue
+def createConsumer(
+    mqService: str,
+    destinations: Optional[list[str]] = None,
+    listenerCls: Optional[Type[stomp.ConnectionListener]] = None,
+) -> StompConsumer:
+    """Create a consumer for the given mqService
+
+    :param mqService: name of the MQService as defined under /Resources/MQServices/
+    :param destinations: list of destinations to listen to. If not defined, take what is
+            defined in the CS
+    :param listenerCls: if defined, given to StompConsumer.addListener
+    """
+    brokerParams = _getBrokerParamsFromCS(mqService)
+
+    host = brokerParams["Host"]
+    port = int(brokerParams["Port"])
+    username = brokerParams["User"]
+    password = brokerParams["Password"]
+
+    csDestinations = brokerParams.pop("destinations", None)
+    destinations = destinations or csDestinations
+    if not destinations:
+        return S_ERROR("Destinations should either be given as parameter or defined in the CS")
+    consumer = StompConsumer(host, port, username, password, destinations)
+
+    if listenerCls:
+        consumer.addListener(listenerCls)
+
+    return consumer
+
+
+@convertToReturnValue
+def createProducer(
+    mqService: str,
+    destination: Optional[str] = None,
+) -> StompProducer:
+    """Create a Producer for the given mqService
+
+    :param mqService: name of the MQService as defined under /Resources/MQServices/
+    :param destination: destination to send to. If not defined, take what is
+            defined in the CS
+    """
+    brokerParams = _getBrokerParamsFromCS(mqService)
+
+    host = brokerParams["Host"]
+    port = int(brokerParams["Port"])
+    username = brokerParams["User"]
+    password = brokerParams["Password"]
+
+    csDestinations = brokerParams.pop("destinations", [])
+
+    if not destination:
+
+        if len(csDestinations) != 1:
+            return S_ERROR("There should be exactly one destination given in parameter or in the CS")
+        destination = csDestinations[0]
+
+    producer = StompProducer(host, port, username, password, destination)
+
+    return producer
```

Comment:
I can't comment on it as it's empty but `src/DIRAC/Resources/MessageQueue/Simple/__init__.py` can be removed now we're Python 3-only.

Reply:
Not sure I understand what you mean. They are still mandatory as far as I understand:

https://docs.python.org/3/tutorial/modules.html

> The __init__.py files are required to make Python treat directories containing the file as packages. This prevents directories with a common name, such as string, unintentionally hiding valid modules that occur later on the module search path. In the simplest case, __init__.py can just be an empty file, but it can also execute initialization code for the package or set the __all__ variable, described later.

Reply:
Indeed I've muddled some things up.

(Specificaly namespace packages which do work but have slightly different behaviour:

```bash
$ mkdir -p a/b/c
$ echo 'print("hello world")' > a/b/c/d.py
$ python -c 'import a.b.c.d'
hello world
```
)